Dockerfile

```diff
@@ -40,5 +40,9 @@ COPY --from=builder /usr/lib/x86_64-linux-gnu/libhwloc.so* /lib/
 COPY --from=builder /usr/lib/x86_64-linux-gnu/libnuma.so* /lib/
 COPY --from=builder /usr/lib/x86_64-linux-gnu/libltdl.so* /lib/
 
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends \
```

Comment:
Looks like this is already being done above on line 19.

Reply:
This one is needed because of the new image being created. Line 19 installs jq inside `builder` but not the final image.

Reply:
Alternatively, the `jq` bin could be copied into the new image.

Reply:
I believe `jq` is written in C and it's possible that it dynamically links in other libs which are expected to be in the environment. But I'll try it out.